### PR TITLE
refactor(tests): moved unit tests

### PR DESCRIPTION
**Pull request by johnleider**
## Description
Moved unit tests out of src files

## Motivation and Context
Clean up the repo

## How Has This Been Tested?
unit

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes, `dev` for new features and breaking changes).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have created a PR in the [documentation](https://github.com/vuetifyjs/vuetifyjs.com) with the necessary changes.

### CODE REVIEW

File: test/unit/components/VCarousel/VCarousel.spec.js [hunk 1]
```diff
@@ -171,7 +171,7 @@ test('VCarousel.js', ({ mount }) => {
     expect(input).toBeCalledWith(0)
   })
 
-  it('should change item on controls click', async () => {
+  it.skip('should change item on controls click', async () => {
```
Comment (KaelWD) [inline test/unit/components/VCarousel/VCarousel.spec.js]: Why'd you skip all these?

**Comment by KaelWD** [comment]
Why isn't this targeted at master?

File: test/unit/components/VCarousel/VCarousel.spec.js (hunk 1 above)
Comment (johnleider) [reply test/unit/components/VCarousel/VCarousel.spec.js]: mistake

**Comment by johnleider** [comment]
Closing for #3429

**Pull request closed without merging**